internal/mongodoc/doc.go

```diff
@@ -60,6 +76,9 @@ type Entity struct {
 	Size int64
 
 	// BlobName holds the name that the archive blob is given in the blob store.
+	// For multi-series charms, there is also a second blob which
+	// stores a "zip-suffix" that overrides metadata.yaml.
+	// This is named BlobName + ".pre-v5-suffix".
```

Comment:
This is not true, it seems to me that it is named BlobName + ".suffix". And perhaps it should change to what we describe here.